src/instructlab/model/backends/backends.py

```diff
@@ -255,26 +255,37 @@ def ensure_server(
         if backend == VLLM:
             # TODO: resolve how the hostname is getting passed around the class and this function
             vllm_server_process = server_process_func(port, background)
-            logger.info(f"Starting a temporary vLLM server at {temp_api_base}")
+            logger.info("Starting a temporary vLLM server at %s", temp_api_base)
```

Comment:
Why these kind of format changes?

Reply:
It avoids the string interpolation if the logging for that level isn't enabled